clang/test/Analysis/initializer.cpp

```diff
@@ -254,6 +254,224 @@ void foo() {
 }
 } // namespace CXX17_aggregate_construction
 
+namespace newexpr_init_list_initialization {
```

Comment:
btw why do we need NewExprs for these? Would the same initialization rules apply for regular variables with automatic storage-duration?